app/frontend/src/submissions/components/SubmissionForm/PersonResponsible.vue

```diff
@@ -178,6 +245,14 @@ export default {
       // keep driller name disabled & set to "person responsible", or leave it enabled and blank
       this.drillerNameInput = (this.personResponsible && this.drillerSameAsPersonResponsible) ? this.personResponsible.name : ''
     }
+  },
+  beforeCreate () {
```

Comment:
I think you should use the `created` hook to be safe; we might be relying on network latency to make `this.persons = response.data` work, and any non-async code added here later could cause errors.